src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -130,6 +130,10 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
     await connection.start();
   } catch (ex) {
     unhandledError(connection, ex, logger);
+
+    if (ex.message.includes('UnsupportedTransportWebSocketsError')) {
```

Comment:
Does this new string appear in the console if a user used SignalR without Blazor? I don't know if we want that.

Reply:
Yes, the `Error.toString()` function outputs `name: message`. So instead of `Error: message` we see `UnsupportedTransportWebSocketsError: message` in the console output. I believe this should be fine?

Reply:
@BrennanConroy ?

Reply:
This one is interesting. Are we taking dependency on the error message then? I don't think that'll be a strong enough contract. Did we talk to the SignalR team to check whether other indicators we can utilize for this purpose?

Reply:
> Are we taking dependency on the error message then?

Not entirely. We're taking a dependency on the `Error.name`. The exception being caught here contains nested exceptions within. One oof those nested exceptions has the `UnsupportedTransportWebSocketsError` we're interested in. 

> Did we talk to the SignalR team to check whether other indicators we can utilize for this purpose?

I believe they're okay with this approach:
https://github.com/dotnet/aspnetcore/pull/34644#discussion_r675724716

Reply:
[This](https://github.com/dotnet/aspnetcore/pull/34644/files#diff-757d83bcf707b34888c824c952e4717638803af7b5372f6c5da1eb838f1a5814R396) message changes as well: 

`Error: ${endpoint.transport} failed: ${ex}` -> `FailedToStartTransportError: ${endpoint.transport} failed: ${ex}`

Reply:
Updated with a new approach (3bb93eb26c) which doesn't change the error message displayed in the console log.